### PR TITLE
Use jenkins credentials

### PR DESCRIPTION
**Pull request by rustyjux**
Retrieve credentials stored in Jenkins with the credentials-binding plugin (already installed) rather than passing as a parameter (which outputs password as plain text to console)

Test build - https://cis.apps.gov.bc.ca/int/job/waops/job/AGO%20ALL_GOV%20Group%20Update/100/console

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -19,8 +19,9 @@ node('imf2public') {
                      bat '''
                     set TEMP=%WORKSPACE%
                     set TMP=%TEMP%
-                    
-                    %PYTHONPATH%python.exe ago-group-update.py -user %agouser% -pwd %agopassword% -group %group%
+                    set AGOPASSWORD=jenkins.model.Jenkins.instance.getCredentials().getById("d9d435e2-8acb-4698-bc74-2a1729472f1a").getPassword()
```
Comment (michaelpnelson) [inline Jenkinsfile]: Does this need to be an environment variable? I'd assume that exposes the pwd in the system running this - if someone else knew the name of the env var they might be able to get its value. A local variable would be safer.

File: Jenkinsfile [hunk 2]
```diff
@@ -34,11 +35,11 @@ node('imf2public') {
...
 
-def notifyFailed() {
-    emailext (
-        subject: "FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]'",
-        body: """<html><body><p>FAILED: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]':</p>
-            <p>Check console output at "<a href="${env.BUILD_URL}">${env.JOB_NAME} [${env.BUILD_NUMBER}]</a>"</p></html></body>""",
-        to: 'datamaps@gov.bc.ca'
-    )
-}
+// def notifyFailed() {
```
Comment (michaelpnelson) [inline Jenkinsfile]: Was this commented-out code left by mistake? If not I'd add a comment above saying why it's commented out

**Review by michaelpnelson — COMMENTED** [review]
Just a few questions

File: Jenkinsfile (hunk 1 above)
Comment (rustyjux) [reply Jenkinsfile]: Firstly, sorry I should have squashed these commits, I didn't use this code - I actually went about it this way - https://github.com/bcgov/ago-all-gov-group-update-tool/pull/7/commits/7d4dc0d67a816cff0139e2bfac1ec79131f64b21

2nd, good question. The password is still an environment variable there, but I'm not sure how accessible it is. e.g. echo %AGOPASSWORD% in the batch script just sent ***** to the console

File: Jenkinsfile (hunk 2 above)
Comment (rustyjux) [reply Jenkinsfile]: Again, my bad not squashing. I commented out to avoid notifications during testing. it's uncommented.

**Review by michaelpnelson — APPROVED** [review]
This looks fine to me.

Note to self: next time review code using the "Files changed" tab instead of "Commits" tab so I can see all changes at once :) (Russell, no need to squash changes when making a PR, though I like to squash changes when I merge a PR so the commit history looks cleaner.)

**Comment by rustyjux** [comment]
Ah yes, Files changed does the trick.

**Pull request merged**